packages/examples/fortune-v2/job-launcher/server/src/currency/currency.service.ts

```diff
@@ -0,0 +1,17 @@
+import { HttpService } from "@nestjs/axios";
+import { Injectable } from "@nestjs/common";
+import { firstValueFrom } from "rxjs";
+
+@Injectable()
+export class CurrencyService {
+  constructor(private readonly httpService: HttpService) {}
+
+  public async getHMTPrice(amount: number, currency: string) {
```

Comment:
Coingeko free api e has some limitations a rate limit of 10-30 calls/minute. What do you think to use cron method and update all rates each hour for example?
`https://api.coingecko.com/api/v3/coins/human-protocol`